timetracker-backend/src/main/java/by/ts/tt/ApplicationConfig.java

```diff
@@ -120,6 +121,11 @@ public void onLogoutSuccess(HttpServletRequest request, HttpServletResponse resp
         };
     }
 
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
```

Comment:
remove wildcard CORS
```suggestion
    public void addCorsMappings(CorsRegistry registry) {
```